Packs/CommonPlaybooks/Playbooks/playbook-Search_Endpoints_By_Hash_-_Generic_V2.yml

```diff
@@ -34,34 +34,26 @@ tasks:
     quietmode: 0
   "5":
     id: "5"
-    taskid: c185447e-549c-476e-8477-98c461dadd16
+    taskid: e540cdba-a994-4570-879c-17357fcbd5bf
     type: playbook
     task:
-      id: c185447e-549c-476e-8477-98c461dadd16
+      id: e540cdba-a994-4570-879c-17357fcbd5bf
       version: -1
-      name: Search Endpoints By Hash - CrowdStrike
-      description: Hunt for endpoint activity involving hash and domain IOCs, using Crowdstrike Falcon Host
-      playbookName: Search Endpoints By Hash - CrowdStrike
+      name: CrowdStrike Falcon - Search Endpoints By Hash
+      playbookName: CrowdStrike Falcon - Search Endpoints By Hash
       type: playbook
       iscommand: false
+      description: ''
       brand: ""
     nexttasks:
       '#none#':
       - "12"
     scriptarguments:
-      MD5Hash:
-        complex:
-          root: inputs
-          accessor: MD5Hash
-      SHA1Hash:
+      FileSha256:
         complex:
-          root: inputs
-          accessor: SHA1Hash
-      SHA256Hash:
-        complex:
-          root: inputs
-          accessor: SHA256Hash
+          root: inputs.SHA256Hash
```

Comment:
since this is a complex one, add a unique transformer here to reduce the number of calls per hash. Pls also ensure that the value is from context ( sometimes that is changed to value -> instead of "from previous tasks").

Reply:
Done.